src/test/java/api/support/builders/ItemBuilder.java

```diff
@@ -1,241 +1,240 @@
-package api.support.builders;
```

Comment:
Do you know why this entire file is showing as changed?

Reply:
I have no clue. After so many times trying to resolve conflicts Git can't even comprehend the differences.  For this commit I had no reason to change this file at all. It got changed due to merging with master, if any.

Reply:
Ok, I'm going to revert the file to the version from master.